### ISSUE TITLE
Animation Curve View doesn't update properly

### ISSUE THREAD

**Issue by HannulaTero** [YoYoGames/GameMaker-Bugs#4552]
### Description
In the beta IDE v2024.200.0.475, Runtime v2024.200.0.490, it seems that animation curve view doesn't update properly when using predefined curves. In the view it does appear create new points, but the line doesn't follow those, and secondly the list of points is not updated. Only after opening and closing curve editor shows new points on the list, but the curve view doesn't reflect that, the all points are shown, but the line follows only first and last.

### Steps To Reproduce
1) Create new Animation Curve asset.
2) Set second point height to 1, points are then [0,0] and [1,1].
3) Select curve from library, for example Bezier - Elastic.
4) Now points are created on the view, but curve doesn't follow them. Also points are not updated on point-list.

[feee40ea-fe5d-4b1c-a33a-2fcf93c0d417](https://api.gamemaker.io/api/github/downloads/feee40ea-fe5d-4b1c-a33a-2fcf93c0d417)

**Comment by YYDan** [comment]
Duplicate of https://github.com/YoYoGames/GameMaker-Bugs/issues/4541
